kpi/deployment_backends/base_backend.py

```diff
@@ -172,18 +172,19 @@ def bulk_update_submissions(
             for path, value in update_data.items():
                 edit_submission_xml(xml_parsed, path, value)
 
-            kc_response = self.store_submission(
-                user, xml_tostring(xml_parsed), _uuid
+            backend_response = self.store_submission(
```

Comment:
Actually it is handled in [`prepare_bulk_update_response`](https://github.com/kobotoolbox/kpi/blob/06db7915939859cc42e3c12dc0714acc3efe4b25/kpi/deployment_backends/openrosa_backend.py#L923). 
I've tried to keep the same logic we have in KobocatDeploymentBackend (i.e. validating response from KC in `prepare_bulk_update_response`). This could be refactored too in #5056 too if we want to catch errors earlier.